geomesa-hbase/README.md

```diff
@@ -0,0 +1,20 @@
+# GeoMesa HBase/BigTable
+
+HBase/BigTable support is broken out into these modules:
+
+## HBase
+
+### [HBase Data Store](geomesa-hbase-datastore)
+
+The HBase data store module contains the code for using an HBase-backed GeoTools data store.
+
+### [HBase GeoServer Plugin](../geomesa-gs-plugin/geomesa-hbase-gs-plugin)
+
+The GeoServer plugin module builds a bundle containing all of the JARs necessary to use the
+HBase data store in GeoServer.
+
+## BigTable
+
+### [BigTable Data Store](geomesa-bigtable-datastore)
+
```

Comment:
maybe add a line about using a google big-table backed store like you have for hbase